documentation/metrics_api.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: default
+title: "Puppet Server: Metrics API"
+canonical: "/puppetserver/latest/metrics_api.html"
+---
+
+# The Metrics API
+
+Puppet Enterprise includes two optional, enabled-by-default web APIs for
+[Java Management Extension (JMX)](https://docs.oracle.com/javase/tutorial/jmx/index.html)
+metrics, namely
+[managed beans (MBeans)](https://docs.oracle.com/javase/tutorial/jmx/mbeans/).
+
+## Metrics v2
+
+The first uses the [Jolokia](https://jolokia.org) library and is accessible at:
+
+    /metrics/v2/
+
+Since the v2 API is implimented using a large open source metrics library with
+its own extensive documentation, below we will only walk through a very brief
+overview with the most common configuration and queries. Each section includes
+links to Jolokia's comprehensive documentation.
+
+### Configuration
+
+For security reasons only the read operations of the Jolokia interface are
+enabled by default. This includes:
+
+* `read`
+* `list`
+* `version`
+* `search`
+
+Users may change the security access policy by providing their own
+`/etc/puppetlabs/puppetserver/jolokia-access.xml` file that follows the
```

Comment:
Yeah, the default policy file is `classpath:/jolokia-access.xml` which uses the locked-down policy packaged in the JAR. Putting a commented example in the default configuration sounds like a good idea.